index.html

```diff
@@ -621,36 +621,6 @@ <h2>Manifest Processing</h2>
 					<p>The following <a href="https://www.w3.org/TR/pub-manifest/#processing-extension">extension
 							steps</a> are added for Audiobook manifests:</p>
 					<ol id="processing-toc">
```

Comment:
I believe the `id` can be removed.
```suggestion
					<ol>
```